dev_setup.py

```diff
@@ -30,6 +30,9 @@ def write_files(api_key, env_path, total_clients):
             file.write("API_KEY={}".format(api_key) + "\n")
             file.write("ID={}".format(i) + "\n")
             file.write("PYTHONUNBUFFERED=TRUE")
+            file.write(f"AWS_ACCESS_KEY={aws_access_key}")
+            file.write(f"AWS_SECRET_ACCESS_KEY={aws_secret_access_key}")
```

Comment:
```suggestion
            file.write("PYTHONUNBUFFERED=TRUE\n")
            file.write(f"AWS_ACCESS_KEY={aws_access_key}\n")
            file.write(f"AWS_SECRET_ACCESS_KEY={aws_secret_access_key}")
```
Should create a new line when writing into the client env file. Client was crashing due not being able to access the credential keys, adding "\n" and then running dev.setup.py fixed the issue on my system.

Reply:
Good catch Alex, Thank you.